src/index.js

```diff
@@ -88,6 +88,11 @@ async function bundleFunctions(file) {
         compact: true,
       }),
     ],
+    onwarn(msg) {
+      if (msg.code == "UNRESOLVED_IMPORT") {
+        throw new Error(`Could not resolve ${msg.source} module. Please install this locally`);
```

Comment:
We should use `utils.build.failBuild()` instead, so this error is reported as a user error, not a plugin error.

Reply:
Good catch! I'll update this